LightBlue.MultiHost/ViewModel/Role.cs

```diff
@@ -64,6 +64,12 @@ public string ImageSource
         {
             get
             {
+                if (App.Configuration.CustomRunners.Any(x => x.Name == Config.RoleName))
+                {
+                    var customRunner = App.Configuration.CustomRunners.First(x => x.Name == Config.RoleName);
+                    return $"Resources\\{customRunner.Icon}.ico";
+                }
```

Comment:
nit: should be able to do this with one scan of the enumerable

```suggestion
                if (App.Configuration.CustomRunners.FirstOrDefault(x => x.Name.Equals(Config.RoleName, StringComparison.OrdinalIgnoreCase)) is CustomRunner customRunner)
                {
                    return $"Resources\\{customRunner.Icon}.ico";
                }
```

Reply:
To expand on this; `is` will check for null so we know if we get into the body of the `if`, then `customRunner` is not null.